WordPress/Classes/Models/Blog.m

```diff
@@ -646,6 +648,11 @@ - (BOOL)supportsContactInfo
     return [self hasRequiredJetpackVersion:@"8.5"] || self.isHostedAtWPcom;
 }
 
+- (BOOL)supportsLayoutGrid
+{
+    return self.isHostedAtWPcom || self.isAtomic;
```

Comment:
Can we include the FeatureFlag check here instead of in the capabilities?

Reply:
Thanks for the suggestion. I added the FeartureFlag here.